px-context-browser.html

```diff
@@ -232,6 +326,27 @@
         reflectToAttribute: true
       },
 
+      /**
+       * Watch for changes to determine if the context browser has failed to sync
+       * its favorited items with those on the app's server. If true, a
+       * contextual notification will be displayed to inform the user and let
+       * them manually resync.
+       */
+      favoritedSyncFailed: {
+        type: Boolean,
+        value: false,
+        notify: true
+      },
+
+      /**
+       * Watch for changes to determine if the context browser is trying to
```

Comment:
Wonder if we should find a way to combine this with `loading`, or if we should give it a name similar to `filter` <> `favoritedFilter`, e.g. `loading` <> `favoritedLoading`. I think having two properties named `loading` and `syncing` that do different things on the same component is confusing.

Reply:
👍